queries/unison/highlights.scm

```diff
@@ -111,5 +121,6 @@
   "]"
 ] @punctuation.bracket
 
-(test_watch_expression
-  (wordy_id) @keyword.directive)
+(watch_expression) @label
```

Comment:
Is this an intended change of the capture? Can you comment on that?

Reply:
It wasn't! I tried this out because I was unsure about what an watch expression exactly is wrt the mapping to nodes. But  I believe `keyword.directive` is more correct.